src/shogun/classifier/mkl/MKL.h

```diff
@@ -95,6 +95,8 @@ class CMKL : public CSVM
 		 */
 		virtual ~CMKL();
 
+		CSGObject* clone();
```

Comment:
please do one or the other in order to be consistent with the rest of the codebase...